docs/mobile-apps/features/instant-booted-simulators.md

```diff
@@ -0,0 +1,391 @@
+---
+id: instant-booted-simulators
+title: Instant Booted iOS Simulators
+sidebar_label: Instant Booted iOS Simulators
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<p><small><span className="sauceDBlue">Beta</span></small></p>
+
+The speed of the Simulator is a crucial factor that impacts the development workflow of iOS app developers. Instant booted iOS Simulators offer significant advantages over our default Simulators, as they reduce the time it takes to launch the Simulator to test apps to almost zero seconds.
+
+With Instant Booted iOS Simulators, you can quickly switch between two iOS versions and test your app in four configurations to ensure the compatibility and performance of your app.
```

Comment:
With Instant Booted iOS Simulators, you can quickly switch between two iOS versions and test your app in four configurations to ensure its compatibility and performance.

Reply:
```suggestion
With Instant Booted iOS Simulators, you can quickly switch between two iOS versions and test your app in four configurations to ensure its compatibility and performance.
```

Reply:
Thanks @eugenekatsov ,

I've changed it